modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/nodecollection/v0.1.0.json

```diff
@@ -37,8 +37,7 @@
               "name": "Properties",
               "description": "Location of circuit file containing general circuit geometry: cell placement, orientation, mvd3.",
               "editorialNote": "BluePyOpt needs to deal with the cell placement mvd3 file.",
-              "seeAlso": "{{base}}/schemas/bbp/core/entity/v0.1.0/shapes/EntityShape",
-              "class": "nsg:Entity",
+              "node": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape",
```

Comment:
The cell placement data needs to be represented in an entity (so that it can get a unique id) which has a distribution object. 

The node constraint is not yet supported in nexus. That's why we put it as a seeAlso for information. When the node constraint is supported in nexus we will put it back.

Reply:
But there is no `distribution` property in bbp.core.Entity, or is it?

Reply:
Transitively yes (from [nexus/core/instance](https://github.com/BlueBrain/nexus-kg/blob/master/modules/kg-schemas/src/main/resources/schemas/nexus/core/instance/v0.1.0.json))

But we can add it in the bbp/core/entity schema explicitly. Can you add it as part of your pull request.

Reply:
If I understand correctly bbp.core.Entity is not nexus.core.Instance, or is it?

Reply:
It is not nexus.core.Instance. 
nexus.core.Instance is the base schema for all instances in nexus. I suggest you add the distribution shape in the bbp.core.Entity.